var/spack/repos/builtin/packages/fenics/package.py

```diff
@@ -72,6 +72,7 @@ class Fenics(CMakePackage):
     patch('hdf5~cxx-detection.patch', when='@:1.6.0')
 
     patch('header_fix.patch', when='@2019.1.0.post0')
+    patch('byte_order.patch', when='@2019.1.0.post0')
```

Comment:
Can we retrieve the patch from a remote `url` and put a brief comment on why it is needed and what is does above the directive?

Reply:
Sure, done.